### PR TITLE
Conversion example for tensile test 

### PR DESCRIPTION
**Pull request by jehona-kryeziu**

### CODE REVIEW

File: example.zip
Comment (schaarj) [inline example.zip:1]: the way  the group is included does not represent the way it is expected in elabFTW exports, where it is typically coming from `item_links` or sth. similar if I recall correctly
Also, I would name the example file `tensile_test_example.eln` and put it with the Jupyter notebook in a subfolder (e.g. "examples" or "sample_data")

File: tensile_test.ipynb [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+   "source": [
+    "## Data conversion for a tensile test experiment sample"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "343d1a1f-f15b-4dea-820d-e3fb62e23a33",
+   "metadata": {},
+   "source": [
+    "This notebook uses an example of the data which is synthetic, but that fits the format of the data derived from the experiments. The goal is to see how the scripts convert the data from the compressed format to an RDF graph and turtle file."
```
Comment (schaarj) [inline tensile_test.ipynb]: add a little bit more markdown like a heading

**Review by schaarj — CHANGES_REQUESTED** [review]
I would add a little bit more markdown between the code blocks. Furthermore I would try to keep the example code as minimal as possible loggers, and try catch blocks are imho not really needed here

File: example.zip
Comment (jehona-kryeziu) [reply example.zip:1]: now we have the exact format for the json data, and all the files are moved to "examples" subfolder and also the eln file is named accordingly

**Review by schaarj — APPROVED** [review]

**Pull request merged**